src/sentry/data_export/processors/discover.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+
+import logging
+
+from sentry.api.utils import get_date_range_from_params
+from sentry.snuba import discover
+from sentry.models import Project
+
+from ..base import ExportError
+
+logger = logging.getLogger(__name__)
+
+
+ALIAS_FIELDS = [("user", ["user.name", "user.email", "user.username", "user.ip"])]
```

Comment:
You might need something similar for `issue` which is another special discover field alias.

Reply:
I can't seem to find anywhere to convert the `issue.id` to that unique `shortId` text

Reply:
Where would that be?

Reply:
That logic sadly ended up in one of the endpoints. I'd like to clean that up but haven't found the time to do so. https://github.com/getsentry/sentry/blob/df0da0722d626670ccc75b0ae655cfd38e47f4d5/src/sentry/api/bases/organization_events.py#L118-L128

Reply:
@markstory I copied the code over from there, but I know there's a comment to move it to the actual `discover.query` method. Should I do that, or is copying it okay for now?

Reply:
Copying is fine for now. If you could leave a comment on both letting future us know where its sibling lives that would be great.

Reply:
Done!